.yamllint.yaml

```diff
@@ -13,11 +13,10 @@ yaml-files:
 rules:
   document-start:
     level: 'error'
-  indentation:
-    spaces: 'consistent'
-    indent-sequences: true
+  indentation: false
   line-length:
     max: 1200
     level: 'warning'
   quoted-strings:
-    quote-type: 'single'
+    quote-type: any
```

Comment:
As much as I'd prefer to keep this one consistent, the existing codebase isn't, so I guess this is the right way to go.

Reply:
We can also follow-up with a one-off lint PR? I have a slight preference for having a consistent style but am not bothered by it either.